sagas/index.js

```diff
@@ -0,0 +1,7 @@
+import { all } from 'redux-saga/effects';
+import apiCitiesSaga from './apiCitiesSaga';
+import apiSportsSaga from './apiSportsSaga';
+
+export default function* rootSaga() {
+  yield all([apiCitiesSaga(), apiSportsSaga()]);
```

Comment:
Naming should be meaningful. Instead of apiCitiesSaga change it to getCitiesSaga.Similary every where follow same convention